amf-cli/shared/src/test/scala/amf/components/OasComponentLinkingTest.scala

```diff
@@ -60,15 +62,12 @@ class OasComponentLinkingTest extends AsyncFunSuite with Matchers {
 
   private def withComponent(uri: String): Future[Module] = {
     val errorHandler: ErrorHandlerProvider = () => UnhandledErrorHandler
-    OASConfiguration
-      .OAS30()
```

Comment:
Why not OasConfiguration.Oas30Component()?

Reply:
Moved to OasConfiguration